package.json

```diff
@@ -1,30 +1,19 @@
 {
   "name": "@buttercup/generator",
-  "version": "0.0.3",
-  "description": "Password generator for Buttercup",
-  "main": "./lib/index.js",
-  "repository": "git@github.com:buttercup/buttercup-generator.git",
-  "author": "Sallar Kaboli <sallar.kaboli@gmail.com>",
-  "license": "MIT",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
   "scripts": {
-    "format": "prettier --write './lib/**/*.js'",
-    "test": "ava",
-    "precommit": "lint-staged"
-  },
-  "lint-staged": {
-    "./lib/**/*.js": [
-      "prettier --write",
-      "git add"
-    ]
+    "test": "mocha -r test/index.js 'test/unit/**/*.spec.js'"
   },
+  "author": "",
+  "license": "MIT",
   "dependencies": {
-    "mngen": "^1.0.0",
-    "password-generator": "^2.2.0"
+    "deep-freeze": "0.0.1",
+    "random-number-generator": "^1.0.2"
   },
   "devDependencies": {
-    "ava": "^0.23.0",
-    "husky": "^0.14.3",
-    "lint-staged": "^4.3.0",
```

Comment:
Can we add back husky and lint-staged and prettier?

Reply:
Yep! Still WIP sorry 😅

Reply:
Ah looked final to me sorry :D